internal/weakdeps/weakdeps.go

```diff
@@ -11,5 +11,5 @@ import (
 	// Ensure that any program using "github.com/golang/protobuf"
 	// uses a version that wraps this module so that there is a
 	// unified view on what protobuf types are globally registered.
-	_ "github.com/golang/protobuf/proto"
+	_ "google.golang.org/protobuf/proto"
```

Comment:
This change is not appropriate. The comment directly above this import explains exactly why this import is for the deprecated `github.com/golang/protobuf`